tests/Unit/CallPowerTest.php

```diff
@@ -0,0 +1,184 @@
+<?php
+
+namespace Tests\Unit;
+
+use Exception;
+use Tests\TestCase;
+use Illuminate\Support\Facades\Bus;
+use Chompy\Jobs\CreateCallPowerPostInRogue;
+use DoSomething\Gateway\Resources\NorthstarUser;
+
+class CallPowerTest extends TestCase
+{
+    /**
+     * Mock the getUser Northstar Call.
+     *
+     * @return user
+     */
+    public function mockGetUserNSCall()
```

Comment:
Let's move this into the `WithMocks` trait, since it seems like something we'd likely want to use in other tests. And this is super nitpicky, but I'd love to rename it to `mockGetNorthstarUser` rather than abbreviating `NS`! 

🤓

Reply:
+1 to moving this into `WithMocks`!